app/views/categories/infrastructure_and_telecommunications.html.erb

```diff
@@ -0,0 +1,76 @@
+<article id="content" class="content-main">
+<% breadcrumb :category, 'Infrastructure and telecommunications' %>
+<% # n.b. All the below ripped from prototype %>
+
+<header><h1>Infrastructure and telecommunications</h1></header>
+
+<p class="abstract">Australian Government information and services related to
+  infrastructure and telecommunications.</p>
+
+<h2>Infrastructure</h2>
+
+<h4>Regional development</h4>
```

Comment:
I know you took this straight from the prototype but this should be an `<h3>` because it lives inside the `<h2>`